core/src/main/java/tachyon/web/UiFileInfo.java

```diff
@@ -119,7 +127,11 @@ public String getCheckpointPath() {
   }
 
   public String getCreationTime() {
-    return CommonUtils.convertMsToDate(mCreationTimeMs);
+    if (mCreationTimeMs == LocalFileInfo.EMPTY_CREATION_TIME) {
+      return " ";
```

Comment:
Oh I see, good point. Could you change them to be empty string? I think the checkpoint path currently will be empty string if it doesn't exist so we have inconsistencies either way.